Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM alpine:3.18
+FROM alpine:3.20
 ARG VERSION=1.22.1
 
 # Inspiration from https://github.com/gmr/alpine-pgbouncer/blob/master/Dockerfile
 # hadolint ignore=DL3003,DL3018
 RUN \
   # security
   apk add -U --no-cache --upgrade busybox && \
+  # https://github.com/krallin/tini#existing-entrypoint
+  apk add -U --no-cache --upgrade tini --repository=https://dl-cdn.alpinelinux.org/alpine/v3.20/community && \
```

Comment:
is the `--repository` switch necessary if we're already using alpine 3.20?

feels like `tiny` could be added to line 8 instead